releasenotes/notes/30014.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+
+releaseNotes:
+- |
+  **Added** Add an environment variable PILOT_JWT_PUB_KEY_REFRESH_INTERVAL for config the interval of
+  istiod to fetch the jwks_uri for the jwks public key. User can use set the refresh interval with command
+  `--set values.pilot.env.PILOT_JWT_PUB_KEY_REFRESH_INTERVAL=20m` or `=10s` while istioctl installation.
```

Comment:
1. Remove the extra `Add`
2. User can set the refresh interval with command `--set values.pilot.env.PILOT_JWT_PUB_KEY_REFRESH_INTERVAL=<duration>`, the default interval is `20m`.